data/DataType/boolean_toml.yaml

```diff
@@ -0,0 +1,4 @@
+keyword: boolean
+literals:
+  - true
```

Comment:
this doesnt render correctly, and even causes a warning with the new enhanced version of statik